apps/api/src/app/shared/shared.module.ts

```diff
@@ -1,4 +1,3 @@
-/* eslint-disable global-require */
```

Comment:
We should not be using `shared.module` at all and rather import what each modules requires one by one.
This creates a situation where we put everything inside shared module and then we just import the shared module everywhere (also in tests) without caring if we really need everything thats in shared causing various issues.